spec/interactors/users/generate_access_token_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+describe Users::GenerateAccessToken do
+  describe ".call" do
+    user = User.create(
+      email: "user@admin.ru",
+      password: "123456"
+    )
+
+    let!(:interactor) { described_class.new(user: user) }
+
+    it "existince of token" do
+      interactor.run
+
+      expect(interactor.context.access_token).present?
+    end
+
+    context "when context contains the user" do
```

Comment:
What are you check here?

Reply:
after user transfer to interactor i want to check the context. Exactly, this is checking the user presence in the context

Reply:
You check interactor gem, not your current interactor. If you put user to context of course it will be there